project_stage_closed/models/task_stage.py

```diff
@@ -10,21 +11,19 @@ class ProjectTaskType(models.Model):
 
     closed = fields.Boolean(
         help="Tasks in this stage are considered closed.",
-        default=False)
-
+        default=False,
+    )
     # a small hack to avoid having two fields with the
     # same name, but show the field in the view
     # when sale_service is not installed
-    closed_alias = fields.Boolean(
-        string='Closed', related='closed')
+    closed_alias = fields.Boolean(string='Closed', related='closed')
     closed_alias_visible = fields.Boolean(
-        compute='_compute_closed_alias_visible')
+        compute='_compute_closed_alias_visible',
+    )
 
     @api.multi
     def _compute_closed_alias_visible(self):
         ir_module = self.env['ir.module.module']
         installed = ir_module.search([('name', '=', 'sale_service'),
                                       ('state', '=', 'installed')])
-        visible = not bool(installed)
-        for o in self:
-            o.closed_alias_visible = visible
+        self.update({'closed_alias_visible': not bool(installed)})
```

Comment:
out of curiosity, why is `update` better? the underlying implementation looks very similar.